### ISSUE TITLE
Loading time is slower

### ISSUE THREAD

**Issue by dtantono** [flathub/org.standardnotes.standardnotes#2]
I tried the flatpak package and I found the loading time for start the app is slower than the official snap or Appimage. Is that so?

**Comment by uncivilgentleman** [comment]
It loads fast for me.

**Comment by dtantono** [comment]
it also ask for keyring everytime I reboot my PC. that means I need to enable password reading on the startup like the snap app. isn't?

**Comment by proletarius101** [comment]
> it also ask for keyring everytime I reboot my PC. that means I need to enable password reading on the startup like the snap app. isn't?

Yes. And I don't think there's anything I can do for this project side. It's more like the integration of Flatpak to your OS. So if you can reproduce this issue in the [bitwarden flatpak](https://github.com/flathub/com.bitwarden.desktop/) you may want to ask at https://github.com/flatpak/flatpak
